src/test/resources/xmpp_client_configs.conf

```diff
@@ -0,0 +1,30 @@
+multiple-muc-config {
+  videobridge {
+    apis {
+      xmpp-client {
+        configs {
+          shard-1 {
+            "DISABLE_CERTIFICATE_VERIFICATION" = true
+            "DOMAIN" = auth.brian.jitsi.net
+            "PASSWORD" = "password\\+"
+            "HOSTNAME" = brian.jitsi.net
+            "MUC" = "JvbBrewery@internal.auth.brian.jitsi.net"
```

Comment:
I think `MUC` is obsolete in favour of `MUC_JIDS`. I don't see any code using `MUC`.

Reply:
ok, i can strip that.